org.messerlab.slimgui.yml

```diff
@@ -0,0 +1,47 @@
+app-id: org.messerlab.slimgui
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+  - --talk-name=com.canonical.AppMenu.Registrar # required for global menu
+  - --talk-name=org.freedesktop.Flatpak # allows spawning processes on the host via flatpak-spawn --host
+  - --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig
+command: SLiMgui
+modules:
+  - name: SLiMgui
+    builddir: true
+    buildsystem: cmake
+    config-opts:
+      - -DBUILD_SLIMGUI=ON
+    sources:
+      - type: git
+        url: https://github.com/messerlab/slim.git
+        branch: master
+        tag: v4.0.1
+        commit: 09fcf604a06307cf75098dfa0a2171a9b7852216
+    post-install:
+      # Colourful and Symbolic Application Icons
+      - sed -i '69,166 s/"fill:#[[:alnum:]]*"/"fill:#ffffff"/' ../QtSLiM/icons/AppIcon16.svg
+      - sed -i '62,68 d' ../QtSLiM/icons/AppIcon16.svg
+      - install --mode=0644 -D --no-target-directory ../QtSLiM/icons/AppIcon64.svg /app/share/icons/hicolor/scalable/apps/org.messerlab.slimgui.svg
+      - install --mode=0644 -D --no-target-directory ../QtSLiM/icons/AppIcon16.svg /app/share/icons/hicolor/symbolic/apps/org.messerlab.slimgui-symbolic.svg
+
+      # slim Mimetype
+      # Line 6 in original org.messerlab.slimgui-mime.xml
+      # <generic-icon name="/usr/share/icons/hicolor/scalable/mimetypes/text-slim.svg"/>
+      - sed -i '6 s,usr,app,' ../org.messerlab.slimgui-mime.xml
+      - sed -i '6 s,mimetypes/text-slim.svg,mimetypes/org.messerlab.slimgui-text-slim.svg,' ../org.messerlab.slimgui-mime.xml
```

Comment:
icon name, not icon path.

Reply:
I take it you mean the `org.messerlab.slimgui-mime.xml` file, which `sed` modifies in-place because there are differences between Flatpak packaging and portable FreeDesktop packaging (for DEB, RPM, etc.; ergo blocking further upstream changes [unless things have changed since then]), should only have the icon name in the `generic-icon` tag name property. I'll push a change to the branch and if that's all good then we can resolve this and the related `install` conversation.